Metro-Records/la-metro-councilmatic
# Allow for notices on all event pages

## Overview

This branch replaces the hard coded public comment instructions with wagtail-editable objects from an `EventNotice` model. These objects will render according to conditions that Metro can select for each notice, and those condition options came from the conditions that were present within the `event_header_*_public_comment` templates. There are two sets of conditions: the status of the broadcast, and its public comment setting.

I've also updated the fixtures with a set of event notices that match what metro currently has on the site.

- Connects #1177

## Testing Instructions
** Note: This PR requires a database migration, the review app will not work and this will have to be tested locally.
 * Pull down this branch and run migrations
   * You may instead run an event scrape if needed, and this would run migrations
 * To have a set of EventNotices ready, load in the cms content with the `load_content` management command
 * Create a super user if needed and log in to the cms `localhost:8001/cms`
   * hannah's from the staging site is in the fixture
 * Head to Event Notices and confirm they have loaded correctly
   * Also confirm that the About Us page content is still intact
 * Make some notices for events that exist for you.
   * Confirm that they render as expected on event detail pages, conditions and all
